vita3k/renderer/src/creation.cpp

```diff
@@ -35,6 +35,34 @@
 
 namespace renderer {
 
+void State::game_start(const char *base_path, const char *title_id, const char *self_name) {
+    this->base_path = base_path;
+    this->title_id = title_id;
+    this->self_name = self_name;
+
+    static const std::string tearaway_games[] = {
```

Comment:
Wouldn't it be better to store this in some kind of .xml file and then call it so that it can be edited and added on to easily? This would be helpful for anything like this in the future. idk just a thought